### PR TITLE
Makes vampire less likely to fire at roundstart (especially on lower populations).

### PR DESCRIPTION
**Pull request by Irradiation**
[balance][role]
I understand that Vampire more or less rolls often because apart from Traitor, it's just about the only other antag that has a "low" cost to fire, so its current weight doesn't matter as much in the grand scheme of things. That said, I'm not a big fan of every round or two in a row containing 1+ vampire instead of something else.

-Weight changed from 3 to 2.
-Cost changed from 18 to 25
-Theat cost changed from (80,60,40,20,20,10,10,10,10,10) to (80,60,50,30,20,10,10,10,10,10)

:cl:
 * tweak: Vampire is less likely to roll at roundstart and moreso on lower populations (15 or less).

### CODE REVIEW

**Comment by Sooose** [comment]
Why less likely to roll at round start? Vampires aside from round start are terrible.

**Comment by Irradiation** [comment]
Because unless my Git-fu is horrible, vampire only launches at roundstart, hence the affected ruleset.

**Comment by Kurfursten** [comment]
Irradiation is correct, there is no latejoin or midround vampire ruleset (only the stone mask)
>I'm not a big fan of every round or two in a row containing 1+ vampire instead of something else

The reasons behind this are much more systemic than what you're fixing here. I should point out that in general people have been okay with pushbacks against nearly every type of antagonist very recently and this is entwined with why we see so much vampire. It's because we're not seeing as many ops, malf, revs, plus ling and revsquad are out entirely. Vampire isn't really my favorite ruleset but at a certain point we're only ever going to see traitor and extended.

In short, if you want to see less vampire the real solution isn't cutting vamp weights it's buffing all team mode weights.

**Comment by Irradiation** [comment]
One PR at a time, but I'm curious about buffing Raging Wiznerds after the heavy nerf they ate (Less likely to happen than New Cops)
